src/main/java/org/LogAggregatorTool/dto/LogAggregatorAuditData.java

```diff
@@ -0,0 +1,80 @@
+package org.LogAggregatorTool.dto;
+
+public class LogAggregatorAuditData {
+    private String pathToTheFolder;
+    private int numberOfLogFiles;
+    private String namesOfLogFiles;
+    private String dateTime;
```

Comment:
Why cant you use Lombok?

Reply:
Lombok dependency is not working in my system.